packages/labs/tsserver-plugin/src/lib/lit-language-service.ts

```diff
@@ -185,6 +203,44 @@ export const makeLitLanguageService = (
       }
       return undefined;
     }
+
+    #getElementClassType(tagname: string) {
+      const checker = this.#analyzer.program.getTypeChecker();
+      // Use the type checker to get the symbol for the ambient/global
+      // HTMLElementTagNameMap type.
+      const tagNameMapSymbol = checker.resolveName(
+        'HTMLElementTagNameMap',
+        undefined,
+        typescript.SymbolFlags.Interface,
+        false
+      );
+
+      if (tagNameMapSymbol?.declarations?.[0]) {
```

Comment:
This if check condition looks odd to me, can you just to:

```suggestion
      if (tagNameMapSymbol) {
```